<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=gb2312">
<title>Drug-Target Interaction Prediction</title>
	    <script>
	    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
	    ga('create', 'UA-38645311-2', 'auto');
	    ga('send', 'pageview');
	    </script>
</head>

<body>
	<table width="800" align="center">
		<tr>
		  <td colspan="2" align="center"><h1>Predicting Drug-Target Interaction for New Drugs Using 
	      Enhanced Similarity Measures and Super-Target Clustering</h1></td>
		</tr>
		<tr><td colspan="2" align="center"><h3>Jian-Yu Shi <sup>a</sup>, Siu-Ming Yiu <sup>b</sup>, Yiming Li <sup>c</sup>, Henry C. M. Leung <sup>b</sup>, Francis Y. L. Chin <sup>*b</sup></h3></td>
		</tr>
		<tr><td colspan="2" align="center">
		  <h5><sup>a</sup> School of Life Sciences, Northwestern Polytechnical University, No.127, Youyi Road West, Xi'an, Shaanxi, China, 710072	  	</h5>
		    <h5><sup>b</sup> Department of Computer Science, The University of Hong Kong, Pokfulam Road, Hong Kong		      </h5>		    
			<h5><sup>c</sup> Department of Psychiatry, The University of Hong Kong, Pokfulam Road, Hong Kong </h5></td></tr>
		<tr><td height="30" colspan="2"></td></tr>
		<tr>
		  <td colspan="2" align="justify"><strong>Abstract</strong>: Predicting drug-target interaction using computational approaches is an important step in drug 
		    discovery and repositioning. To predict whether there will be an interaction between a drug and a 
		    target, most existing methods identify similar drugs and targets in the database. The prediction 
		    is then made based on the known interactions of these drugs and targets. This idea is promising. 
		    However, there are two shortcomings that have not yet been addressed appropriately. Firstly, most 
		    of the methods only use 2D chemical structures and protein sequences to measure the similarity of 
		    drugs and targets respectively. However, this information may not fully capture the characteristics 
		    determining whether a drug will interact with a target. Secondly, there are very few known interactions, 
		    i.e. many interactions are "missing"; in the database. Existing approaches are biased towards 
		    known interactions and have no good solutions to handle possibly missing interactions which affect
		    the accuracy of the prediction. In this paper, we enhance the similarity measures to include nonstructural 
		    (and non-sequence-based) information and introduce the concept of a "super-target"; to 
		    handle the problem of possibly missing interactions. Based on evaluations on real data, we show 
		    that our similarity measure is better than the existing measures and our approach is able to achieve 
	      higher accuracy than the two best existing algorithms, WNN-GIP and KBMF2K.</td>
		</tr>
		<tr><td height="30" colspan="2"></td></tr>
		
		<tr >
			<td colspan="2" align="justify"><p>&nbsp;</p>
			  <p><strong>INPUT</strong></p>
			  <p>The  INPUT to our method is the similarities between the new drug and those known  drugs in the datasets .The drug similarity is measured by chemical-structure  alignment and ATC, and the target similarity is measured by protein-sequence  alignment and FC respectively. Both the similarities between the known drugs  and those between the known targets are pre-calculated and built-in.</p></td> 
			
		
		</tr>
		<tr><td colspan="2" align="justify"><p>&nbsp;</p>
		    <p><strong>OUTPUT</strong></p>
		    <p>The  OUTPUT from our method is a set of the confidence scores which denote how  likely this new drug interacts with all known targets respectively.</p>
		    </td>
	  </tr>
		<tr><td colspan="2" align="justify"><p>&nbsp;</p>
		  <p><strong>Computational Methods Involved </strong></p>
		  <p>The computational methods used in our methods include Hattori et al.'s method [1] for drug chemical-structure similarity, the protein-sequence alignment algorithm of Smith and Waterman[2] for protein sequence similarity, the agglomerative hierarchical clustering [3] for building Super-Target and the KNN classifier [4] for performing DTI predictions.		  </p>
		  <p>
					The datasets used for training and evaluating our model were collected by Yamanishi et al [5] and can be downloaded from their webpage (<a href="http://web.kuicr.kyoto-u.ac.jp/supp/yoshi/drugtarget/" target="_blank">http://web.kuicr.kyoto-u.ac.jp/supp/yoshi/drugtarget/</a>). Yamanishi et al. [5]  collected and integrated
		drug-target interactions from the KEGG BRITE [6], BRENDA [7], SuperTarget & Matador
		[8] and DrugBank databases [9]. These datasets were also used as the benchmark datasets for
		comparing the performance of DTI prediction algorithms by subsequent works (e.g. KBMF2K [10]
		and WNN-GIP [11]). The details about how to collect drugs, targets and their interactions, can be
		checked in their original paper [5].		</p>
		  <p>All benchmark DTIs are split into four datasets according to the type of protein targets, including
		    enzyme, ion channel (IC), G protein-coupled receptor (GPCR) and nuclear receptor (NR).
		    The numbers of drugs, targets and their unknown interactions in the corresponding datasets are
		    listed in Table 1. </p>
		  <p>
		  <table width="480" border="1" align="center">
					  <caption>
						Table 1. Four benchmark datasets used in our work
					  </caption>
					  <tr>
						<th scope="row">&nbsp;</th>
						<td width="60">Enzyme</td>
						<td width="60">IC</td>
						<td width="60">GPCR</td>
						<td width="60">NR</td>
					  </tr>
					  <tr>
						<th scope="row">Number of drugs</th>
						<td>445</td>
						<td>210</td>
						<td>223</td>
						<td>54</td>
					  </tr>
					  <tr>
						<th scope="row">Number of targets</th>
						<td>664</td>
						<td>204</td>
						<td>95</td>
						<td>26</td>
					  </tr>
					  <tr>
						<th scope="row">Number of interactions</th>
						<td>2926</td>
						<td>1476</td>
						<td>635</td>
						<td>90</td>
					  </tr>
		  </table>

			</p>
			
	    <p>All ATC codes of the drugs can be found in KEGG (<a href ="http://www.kegg.jp/kegg/drug/" target="_blank">http://www.kegg.jp/kegg/drug/</a>) and all FC codes of the targets can be found in HUGO Gene Nomenclature Committee (<a href ="http://www.genenames.org/" target="_blank">http://www.genenames.org/</a>). </p></td></tr>
		
		<tr>
		<td colspan="2"> <p>&nbsp;</p>
		  <p><strong>Implementation</strong>		  </p>
		  <p>The input of our method includes pairwise drug similarities, pairwise target similarities and known drug-target interactions. The output is the confidence scores which denote how likely the pairs of known targets and new drugs are potential interactions. The whole workflow is illustrated in Fig 1.
	      </p>
		  <table width="600" border="0" align="center">
			  <tr>
				<td align="center"><img src="Workflow.jpg" width="80%" ></td>
			  </tr>
			  <tr>
				<td align="justify">Figure 1: The work
ow of predicting drug-target interaction for new drugs. The whole work
ow contains two phases,the training phase and the testing phase. In the first phase, pairwise similarities {S<sub>d</sub>} between known drugs are used
to build two predicting models for each super-target which is generated by clustering known targets based on pairwise
target similarities {S<sub>t</sub>}, and for each target in the super-target respectively. In the second phase, the similarities
between a new drug dx and known drugs are input into Predicting Model 1 to obtain Score 1 which denotes how
likely dx interacts with super-target stk, and then input into Predicting Model 2 to obtain Score 2 which re
ects how likely dx interacts with the known target tp belonging to stk. The final confidence score of dx connecting with tp is obtained by
the product of Score 1 and Score 2. Sd and St are the pairwise drug similarity and pairwise target similarity.</td>
			  </tr>
		  </table>
		<p><strong>Download</strong>: <a href="DataAndFunctions.zip">SourceCodes</a></p>
		</td>
		</tr>
		
		
		<tr><td height="30" colspan="2"></td></tr>
		<tr>
		<td colspan="2" align="justify"><p><strong>References</strong></p>
		
		  <ol>
		    <li>
		      M. Hattori, Y. Okuno, S. Goto, et al., Development of a chemical structure comparison method
		      for integrated analysis of chemical and genomic information in the metabolic pathways, J. Am.
		      Chem. Soc. 125 (39) (2003) 11853-11865.	        </li>
	        <li>T. F. Smith, M. S. Waterman, Identification of common molecular subsequences, J. Mol. Biol.
	          147 (1) (1981) 195-197.	          </li>
		    <li>J. H. Ward Jr, Hierarchical grouping to optimize an objective function, J. Am. Statist. Assoc.
		      58 (301) (1963) 236-244.		      </li>
		    <li>M.-L. Zhang, Z.-H. Zhou, ML-KNN: A lazy learning approach to multi-label learning, Pattern
		      Recogn. 40 (7) (2007) 2038-2048.		      </li>
		    <li>Y. Yamanishi, M. Araki, A. Gutteridge, et al., Prediction of drug-target interaction networks
		      from the integration of chemical and genomic spaces, Bioinformatics 24 (13) (2008) i232-i240.		      </li>
		    <li>M. Kanehisa, S. Goto, M. Hattori, et al., From genomics to chemical genomics: new developments
		      in KEGG, Nucleic Acids Res. 34 (Database issue) (2006) D354-D357.		      </li>
		    <li>I. Schomburg, A. Chang, C. Ebeling, et al., BRENDA, the enzyme database: updates and
		      major new developments, Nucleic Acids Res. 32 (Database issue) (2004) D431-D433.		      </li>
		    <li>S. G&uuml;nther, M. Kuhn, M. Dunkel, et al., SuperTarget and Matador: resources for exploring
		      drug-target relationships, Nucleic Acids Res. 36 (Database issue) (2008) D919-D922.		      </li>
		    <li> D. S. Wishart, C. Knox, A. C. Guo, et al., DrugBank: a knowledgebase for drugs, drug actions
		      and drug targets, Nucleic Acids Res. 36 (Database issue) (2008) D901-D906.		      </li>
		    <li> M. G&ouml;nen, Predicting drug-target interactions from chemical and genomic kernels using
		      Bayesian matrix factorization, Bioinformatics 28 (18) (2012) 2304-2310.		      </li>
		    <li> T. van Laarhoven, E. Marchiori, Predicting drug-target interactions for new drug compounds
		      using a weighted nearest neighbor profile, PloS One 8 (6) (2013) e66952. </li>
		  </ol></td>
		  
	  </tr>
	</table>
</body>
</html>
